### PR TITLE
xGov-8 - Evil Tools

### PR DESCRIPTION
**Pull request by LoafPickleWW**
Evil Tools is our most devious plot yet to onboard creators and collectors to Algorand. These no-code tools help make repetitive tasks easy and efficient. *Twirls Mustache*

### CODE REVIEW

File: Proposals/xGov-8 [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Experience with Algorand
+All of us have been in the Algorand NFT Space since May 2021. Thurstober Digital Studios has built a full horse racing utiltiy series and our body of work with Evil Tools speaks for itself. 
+
+Website: https://thurstober.com/
+Evil Tools Website: https://evil-tools.thurstober.com/
+
+## Roadmap
+Brrrrrrr Mode:
+A way for users to get away from the Pera limitations of only being able to sign 64 tx at a time.
```
[the remark this answers is not in the document]
Comment (LoafPickleWW) [reply Proposals/xGov-8:35]: Yeah sure. The site does not store any information. We would have a means for a creator to enter their keys and connect right to the SDK. 

With the inclusion of rekey tech, creators can rekey to a burner wallet and use that to enter the seed of the burner, then rekey back to their private keys. For the Airdrop tool, an escrow wallet could be used. I would write out guides and warnings so people feel their comfiest, but they can always sign through Pera if they feel uncomfortable.

**Comment by No-Cash-7970** [comment]
Can you rename the `xGov-8` to `xGov-8.md` so it's easier to read on GitHub?

File: Proposals/xGov-8.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+period: 8
+title: Evil Tools
```
Comment (SudoWeezy) [inline Proposals/xGov-8.md:3]:
```suggestion
title: Evil Tools
```

File: Proposals/xGov-8.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+period: 8
+title: Evil Tools 
+author: Joseph Glenn (LoafPickle) (@LoafPickleWW)
```
Comment (SudoWeezy) [inline Proposals/xGov-8.md:4]:
```suggestion
author: Joseph Glenn (@LoafPickleWW)
```

File: Proposals/xGov-8.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+company_name: Thurstober Digital Studios LLC
+category: Tools
+focus_area: No-Code Tooling for Non-Developers to help onboard people to algorand
```
Comment (SudoWeezy) [inline Proposals/xGov-8.md:7]:
```suggestion
focus_area: User Onboarding
```

File: Proposals/xGov-8.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+open_source: Undecided
+amount_requested: 10,000A
```
Comment (SudoWeezy) [inline Proposals/xGov-8.md:9]:
```suggestion
amount_requested: 10000
```

**Review by SudoWeezy — CHANGES_REQUESTED** [review]
Thanks for your proposal @LoafPickleWW. I have suggested some formatting changes to have a proper structure.

**Comment by LoafPickleWW** [comment]
> Thanks for your proposal @LoafPickleWW. I have suggested some formatting changes to have a proper structure.

Fixed

**Comment by andrew-m-higgs** [comment]
Personally, I would love to see this open-sourced.

**Comment by LoafPickleWW** [comment]
Brrr mode has been renamed to infinity mode and has been built.

Will we finish the rest of the tools before the voting period? Well Evil Tools conquer Algorand? Find out next time on Dragon Ball Z.

**Comment by LoafPickleWW** [comment]
> Personally, I would love to see this open-sourced.

It's not out of the question. We're in heavy development mode at the moment. Once it gets to the point where I'm happy, I'm leaning to OS it.

**Comment by LoafPickleWW** [comment]
Arc 19 collection information tool has been built. 2/7 on roadmap complete. 

Mass Destroy next.

**Review by Fritos2 — APPROVED** [review]

**Comment by LoafPickleWW** [comment]
Both the Mass Asset Destroy and Premium Send tools have been built. 

4/7 Done.

File: Proposals/xGov-8.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+---
+period: 8
+title: Evil Tools
+author: Joseph Glenn (@LoafPickleWW)
+company_name: Thurstober Digital Studios LLC
+category: Tools
+focus_area: User Onboarding
+open_source: No
+amount_requested: 10000
+status: Draft
```
Comment (SudoWeezy) [inline Proposals/xGov-8.md:10]: Remember to change this to "Final" if you want your proposal to go through the next voting session.

Comment (LoafPickleWW) [reply Proposals/xGov-8.md]: > Remember to change this to "Final" if you want your proposal to go through the next voting session.

Done.

**Comment by LoafPickleWW** [comment]
Finished the opt in check and holder features updates.

**Comment by ShaperOfEntropy** [comment]
Definitely a useful project. Good cost-benefit ratio.

My comments:
1) The proposal could be better structured, since it isn't very clear at a quick glance what is being proposed to be built vs. what has already been built.
2) Analysis of competitors / other projects with similar solutions and how the project differs could be included.
3) How will the grant be spent? Will you be selling it on the market? If yes, using DEXs or CEXs? Or will you take a loan against it in DeFi/CeFi? Or will project team member be paid in ALGO?
4) What's the long-term vision for maintenance of the project outcome? How will it make money to be sustainable? Is it meant to become purely community driven, live from donations, etc.

**Comment by nonfungibleab** [comment]
Congratulations! xGov voting session 1 is now closed and your proposal has passed. I need your email to kick off contract signature next week. Would you please send me a DM on Twitter or Discord?

**Pull request merged**